docs/management/api/scan-api.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[scan-api]]
+=== Scan a file or folder
+
+Scan a file or folder on a host running {elastic-defend} for malware.
+
+You must have the `Scan Operations` {kib} privilege in the Security feature as part of your role and at least an Enterprise license to perform this action.
```

Comment:
Maybe remove "...at least..." since `Enterprise` is the highest license a company can have.

Reply:
Done d48f7883bfcd5facae8c638c2da4ad62de495e1b